services/imagery/src/image-store.js

```diff
@@ -180,10 +189,13 @@ export default class ImageStore extends EventEmitter {
     await this._withDb(async (db) => {
       if (id === undefined) {
         id = (await db.run('INSERT INTO images DEFAULT VALUES')).lastID;
+        // Can we simplify this to just one insert statement?
+        await db.run('UPDATE images SET size = ? WHERE id = ?',
+          [metadata.size, id]);
```

Comment:
Yes, it's possible to simplify into this:

```sql
INSERT INTO images(size) VALUES (?);
```

The rest of the columns will be populated with their default values.